android/app/src/debug/AndroidManifest.xml

```diff
@@ -1,7 +1,10 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.example.at_dude">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.example.at_dude">
```

Comment:
This is still `com.example.at_dude`, while the others have been changed to `com.atsign.at_dude`

Reply:
I suggest doing a global search for this to make sure that none have been missed.

Reply:
fixed